composer.json

```diff
@@ -23,7 +23,7 @@
     "require": {
         "php": ">=7.2",
         "monolog/monolog": "^2.1",
-        "guzzlehttp/psr7": "^1.6"
+        "guzzlehttp/psr7": "^1.6|^2.0"
```

Comment:
Hello, thank you for your support. Let's add a step to the ci process of installing different versions - to test each

Reply:
@Yozhef done

Reply:
Thank you for your support